src/standalone/UIKit/InputControls/DateInput.tsx

```diff
@@ -25,30 +75,97 @@ const DateInput = (props: DateInputProps & DatePickerProps) => {
 		},
 		[openInfo]
 	);
+	const handleTextFieldOnChange = useCallback(
+		(event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>) => {
+			handleCursorChange(event);
+
+			if (!onChange) return;
 
+			const formatted = TypeDate.format(event.target.value);
+			setTextValue(formatted);
+
+			const date = new Date(formatted);
+			// If the fields of date are NaN the date is invalid, we should raise an error for this (onError prop)
+			if (formatted.includes("_") || isNaN(date.getTime())) {
+				setErrorHelperText(
+					ccI18n.t(
```

Comment:
this should be overridable (there should be something in the picker props for this label)
`props.invalidDateMessage ?? ccI18n.t(...)`